lib/addons/library.kodi.guilib/project/VS2010Express/post-build.ps1

```diff
@@ -3,7 +3,7 @@ param (
 )
 
 $LIB_INTERFACE = "$ProjectDir\..\..\..\..\..\addons\library.kodi.guilib\libKODI_guilib.h"
-$GENERATED_ADDON_GUILIB = "$ProjectDir\..\..\..\..\..\addons\kodi.guilib\addon.xml"
+$GENERATED_ADDON_GUILIB = "$ProjectDir\..\..\..\..\..\addons\library.kodi.guilib\addon.xml"
```

Comment:
yes i see i went the wrong direction in this PR.
thought it needed to end up in library.kodi.guilib but needs to be kodi.guilib